### PR TITLE
Create rating.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
rating

### CODE REVIEW

File: Components/rating.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.rating-star-input:checked + .rating-star-icon {
+  color: var(--dark-info-color);
+}
+
+.rating-badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: fit-content;
+  font-size: 12px;
```
Comment (shivampandey0) [inline Components/rating.css]: use rem for units greater than 10px.

File: Components/rating.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  line-height: 12px;
+  color: black;
+  background-color: var(--light-secondary-color);
+  padding: 5px;
+}
+.rating-badge-star {
+  color: var(--dark-info-color);
+  width: 12px;
+  height: 12px;
+  margin-right: 4px;
```
Comment (shivampandey0) [inline Components/rating.css:36]: variables can be used for units.

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions

File: Components/rating.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/rating.css]: done

File: Components/rating.css (hunk 2 above)
Comment (Radheshyam1234) [reply Components/rating.css:36]: done

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

**Pull request merged**